### PR TITLE
Use install-depencies.sh in workflow

### PR DESCRIPTION
**Pull request by matthiakl**
Implements https://github.com/widelands/widelands/pull/5481#issuecomment-1204958405
There were actually some errors in the script, so it is a good idea to use it in the workflow.

### CODE REVIEW

**Review by Noordfrees — APPROVED** [review]
:+1:

If this fixes actual errors in the installer script, any reason not to have it for v1.1?

File: install-dependencies.sh [hunk 1]
```diff
@@ -115,73 +119,74 @@ sudo_or_su() {
...
     mingw-w64-i686-glbinding mingw-w64-i686-glew mingw-w64-i686-icu
 
 elif [ "$DISTRO" = "msys64" ]; then
    echo "Installing dependencies for 64-bit Windows..."
-   pacman -S mingw-w64-x86_64-toolchain git mingw-w64-x86_64-cmake \
+   pacman -S $@ mingw-w64-x86_64-toolchain git mingw-w64-x86_64-cmake \
     mingw-w64-x86_64-ninja mingw-w64-x86_64-asio mingw-w64-x86_64-SDL2_ttf \
     mingw-w64-x86_64-SDL2_mixer mingw-w64-x86_64-SDL2_image \
-    mingw-w64-x86_64-glbinding mingw-w64-x86_64-glew mingw-w64-i686-icu
+    mingw-w64-x86_64-glbinding mingw-w64-x86_64-glew mingw-w64-x86_64-icu
```
Comment (tothxa) [inline install-dependencies.sh]: Could these be unified like it was in the workflow file? Maybe define a shell function with $ARCH as argument?

**Comment by matthiakl** [comment]
> If this fixes actual errors in the installer script, any reason not to have it for v1.1?

This is nothing critical, so I thought it is not a must have for v1.1. Of course you can decide to merge it before :wink:

File: install-dependencies.sh (hunk 1 above)
Comment (tothxa) [reply install-dependencies.sh]: Thank you. Also for picking this up. :)

**Pull request merged**